templates/views/index.html

```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
         <div class="row">
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
                 <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                    <p class="__shecodetestimp">This Sca mentoring program was like a breathe of fresh air that I needed
+                        since December 2019. I'm so filled as I joined
+                        and has been growing on my learning path (Php/laravel).
```

Comment:
Change 'and has been growing on my learning path (Php/laravel)' to 'and I have been growing on my learning path (PHP/Laravel).'